docs/docs/integrations/toolkits/slack.ipynb

```diff
@@ -151,7 +151,7 @@
    "outputs": [],
    "source": [
     "agent_executor.invoke(\n",
-    "    {\"input\": \"Send a greeting to my coworkers in the #general channel.\"}\n",
+    "    {\"input\": \"Send a greeting to my coworkers in the #general channel. Note use `channel` as key of channel id, and `message` as key of content to sent in the channel.\"}\n",
```

Comment:
Why add this cell and not execute it?

Reply:
Actually this is keeping unchanged with previous version. I just copy&paste the code verified runnable here. I can execute it if preferred.

Reply:
ah I see, my mistake. I reverted that commit and re-did the lint. Thanks very much for this!